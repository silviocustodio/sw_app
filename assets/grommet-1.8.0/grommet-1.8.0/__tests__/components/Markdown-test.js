// (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP

import React from 'react';
import renderer from 'react-test-renderer';

import Markdown from '../../src/js/components/Markdown';

// needed because this:
// https://github.com/facebook/jest/issues/1353
jest.mock('react-dom');

describe('Markdown', () => {
  it('has correct default options', () => {
    const component = renderer.create(
      <Markdown content='test'
        components={{
          p: { props: { className: 'testing', size: 'large' } }
        }} />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
