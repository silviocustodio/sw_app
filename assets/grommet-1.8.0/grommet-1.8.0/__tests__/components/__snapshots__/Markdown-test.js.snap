// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown has correct default options 1`] = `
<p
  className="grommetux-paragraph grommetux-paragraph--large testing"
>
  test
</p>
`;
